test/e2e-tests-upgrade.sh

```diff
@@ -41,41 +62,18 @@ header "Setting up environment"
 set +o errexit
 set +o pipefail
 
-# First, we will verify if Scenario 1 works.
```

Comment:
can you help me understand why the logic for scenarios 1 and 2 is being removed?

Reply:
That was changed when I wanted to provide script as an example for trying out the results at https://github.com/tektoncd/pipeline/pull/6690#discussion_r1211735679 which should not be a change made in the PR itself, restored.